spec/support/database.rb

```diff
@@ -1,5 +1,6 @@
 require "logger"
 require "active_record"
+require "erb"
```

Comment:
What is `erb` doing here?

Reply:
https://github.com/ManageIQ/activerecord-virtual_attributes/pull/35/files#diff-08d5f77d01f8f94063c830ce47ebfc74L29

This was throwing an error before without the added `require`

Huh, but looks like it is working now.
So I'll remove it.

... confused ...